lib/src/widgets/country_selector/country_list.dart

```diff
@@ -38,7 +39,8 @@ class CountryList extends StatelessWidget {
     required this.noResultMessage,
     this.scrollController,
     this.scrollPhysics,
-    this.showDialCode = true,
+    this.showDialCode = false,
```

Comment:
why did you change the default ?

Reply:
Oh Sorry, I was play around with params and forgot to reset this

Reply:
Reopening new pull request

Reply:
Fixed by adding new commit